# Activity_Week09

#### Dataset used : Advertising.csv
#### 200 rows × 5 columns

In [1]:
import pandas as pd
import numpy as np

In [2]:
data=pd.read_csv('Advertising.csv')
data.head()

Unnamed: 0     TV  Radio  Newspaper  Sales
0           1  230.1   37.8       69.2   22.1
1           2   44.5   39.3       45.1   10.4
2           3   17.2   45.9       69.3    9.3
3           4  151.5   41.3       58.5   18.5
4           5  180.8   10.8       58.4   12.9

In [3]:
d1 = data.drop(data.columns[0], axis=1)
d1.head()

TV  Radio  Newspaper  Sales
0  230.1   37.8       69.2   22.1
1   44.5   39.3       45.1   10.4
2   17.2   45.9       69.3    9.3
3  151.5   41.3       58.5   18.5
4  180.8   10.8       58.4   12.9

## 1. Explain the dataset

In [4]:
d1.shape

(200, 4)

In [5]:
d1.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 200 entries, 0 to 199
Data columns (total 4 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   TV         200 non-null    float64
 1   Radio      200 non-null    float64
 2   Newspaper  200 non-null    float64
 3   Sales      200 non-null    float64
dtypes: float64(4)
memory usage: 6.4 KB


#### There are 200 entries in the given dataset. No null values are present in the given dataset.

## 2. Check the summary statistics and discuss the max, min, avg, median, and percentiles.

In [6]:
d1.describe()

TV       Radio   Newspaper       Sales
count  200.000000  200.000000  200.000000  200.000000
mean   147.042500   23.264000   30.554000   14.022500
std     85.854236   14.846809   21.778621    5.217457
min      0.700000    0.000000    0.300000    1.600000
25%     74.375000    9.975000   12.750000   10.375000
50%    149.750000   22.900000   25.750000   12.900000
75%    218.825000   36.525000   45.100000   17.400000
max    296.400000   49.600000  114.000000   27.000000

#### The above table shows the summary statistics for the given dataset. There are 200 entries. The mean value for Sales is 14.022500 and the standard deviation is 5.217457. The maximum value for Sales is 27.0 and the minimum value is 1.60.  The prcentile values for Sales are as follows :
#### 25% - 10.375000
#### 50% - 12.900000 (MEDIAN)
#### 75% - 17.400000

## 3. The manager wants to find out whether the same amount was spent for the three advertisements (TV , Radio and Newspaper). Comment on your findings.

## Using One-way ANOVA :

#### Level of significance, alpha = 0.05
#### Null Hypothesis, H0 : Same amount was spent for the three advertisements (TV , Radio and Newspaper)
#### Alternate Hypothesis, Ha : Different amount was spent for the three advertisements (TV , Radio and Newspaper)

In [7]:
from scipy.stats import f_oneway

In [8]:
f_oneway(d1['TV'], d1['Radio'], d1['Newspaper'])

F_onewayResult(statistic=358.8514595342597, pvalue=4.552931539744962e-103)

## Conclusion :

#### Here, p-value(4.552931539744962e-103) is less than the alpha value (0.05). Hence we reject the null hypotheis and it can be concluded that different amount was spent for the three advertisements (TV , Radio and Newspaper).